pkg/trace/agent/agent.go

```diff
@@ -318,6 +321,26 @@ func (a *Agent) ProcessStats(in pb.ClientStatsPayload, lang string) {
 				TopLevel: float64(b.Hits),
 				Value:    float64(b.Duration),
 			}
+			if hitsSummary, errorSummary, err := quantile.DDToGKSketches(b.HitsSummary, b.ErrorSummary); err == nil {
```

Comment:
Nit: can we reverse the `if` branches? It's usually `err != nil` first. Also, the variable names seem unnecessarily long, would you mind shortening?
```suggestion
			if hits, errors, err := quantile.DDToGKSketches(b.HitsSummary, b.ErrorSummary); err == nil {
```